parachain/pallets/ethereum-beacon-client/src/merkleization.rs

```diff
@@ -121,6 +156,18 @@ pub fn hash_tree_root_execution_header<
 	hash_tree_root(ssz_execution_payload)
 }
 
+pub fn hash_tree_root_execution_header_capella<
```

Comment:
I also feel like we can have one version of `hash_tree_root_execution_header` that takes the execution header type as a generic parameter.